app/build.gradle

```diff
@@ -33,24 +36,24 @@ dependencies {
 
     // Support libs
     def support_lib_version = '1.0.0'
-    implementation "androidx.appcompat:appcompat:1.0.2"
+    implementation "androidx.appcompat:appcompat:1.1.0"
     implementation "com.google.android.material:material:$support_lib_version"
     implementation "androidx.recyclerview:recyclerview:$support_lib_version"
     implementation "androidx.cardview:cardview:$support_lib_version"
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
 
     // Room
-    def room_version = "2.0.0"
+    def room_version = "2.1.0"
     implementation "androidx.room:room-runtime:$room_version"
-    annotationProcessor "androidx.room:room-compiler:$room_version"
+    kapt "androidx.room:room-compiler:$room_version"
```

Comment:
Change to: `annotationProcessor "androidx.room:room-compiler:$room_version"`

Reply:
No, Room requires `kapt` for annotation processing when using Kotlin. The default `annotationProcessor` doesn't work with Kotlin code. So this is to be kept.

Reply:
@AniVerma17 Room's models are written in Java. Are you sure that room is asking for kapt?

Reply:
> @AniVerma17 Room's models are written in Java. Are you sure that room is asking for kapt?

They have been converted to Kotlin, since they were also among the POJO classes. So Room requires `kapt` for processing its annotations on those classes, else results in build error.

Reply:
You are right. Almost missed those files.
 :shipit:

Reply:
I tested on my local machine and didn't get any errors. I think because I already got those files in my cache.